pkg/apis/autoscaling/v1/clusterautoscaler_types.go

```diff
@@ -44,11 +57,22 @@ type ClusterAutoscalerList struct {
 }
 
 type ResourceLimits struct {
+	// Maximum number of nodes in all node groups.
+	// Cluster autoscaler will not grow the cluster beyond this number.
 	// +kubebuilder:validation:Minimum=0
-	MaxNodesTotal *int32         `json:"maxNodesTotal,omitempty"`
-	Cores         *ResourceRange `json:"cores,omitempty"`
-	Memory        *ResourceRange `json:"memory,omitempty"`
-	GPUS          []GPULimit     `json:"gpus,omitempty"`
+	MaxNodesTotal *int32 `json:"maxNodesTotal,omitempty"`
+
+	// Minimum and maximum number of cores in cluster, in the format :.
+	// Cluster autoscaler will not scale the cluster beyond these numbers.
+	Cores *ResourceRange `json:"cores,omitempty"`
+
+	// Minimum and maximum number of gigabytes of memory in cluster, in the format :.
+	// Cluster autoscaler will not scale the cluster beyond these numbers.
+	Memory *ResourceRange `json:"memory,omitempty"`
+
+	// Minimum and maximum number of different GPUs in cluster, in the format <gpu_type>::.
+	// Cluster autoscaler will not scale the cluster beyond these numbers. Can be passed multiple times.
```

Comment:
is this correct `in the format <gpu_type>::.` ?

Reply:
https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-are-the-parameters-to-ca

Reply:
changed to `<gpu_type>:<min>:<max>`